test/lit/passes/cfp-rmw.wast

```diff
@@ -0,0 +1,898 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: foreach %s %t wasm-opt --remove-unused-names --cfp -all -S -o - | filecheck %s
+
+;; (remove-unused-names is added to test fallthrough values without a block
+;; name getting in the way)
```

Comment:
This should not be needed, as the parser does not add new block names (since we fixed that issue).